.github/workflows/maintenance.yml

```diff
@@ -2,8 +2,10 @@ name: Perform automated repository maintenance
 
 on:
```

Comment:
Not sure an empty section is valid.
It may be better to comment out the whole section block.

Reply:
The section is not actually empty 🙃, [the next line](https://github.com/zappa/Zappa/pull/1300/files#diff-6eb5da11d7badb128a54d987e293ca9eec3cbb461d01a0abeae2d7a09675aefbL4) (`workflow_dispatch`) is part of this `on` section, and these two lines were intentionally _not_ commented-out because `workflow_dispatch` is the "event trigger" that allows a workflow to be run manually[^1].

The evidence that these lines work as expected can already be seen when viewing [the workflow from the Actions tab](https://github.com/zappa/Zappa/actions/workflows/maintenance.yml), where we can see that the **`Run workflow`** button which allows the workflow to be run manually[^2] is successfully appearing, along with a message that confirms that `workflow_dispatch` is "wired" correctly, allowing the button to appear:
>This workflow has a `workflow_dispatch` event trigger.

![image](https://github.com/zappa/Zappa/assets/31746850/69b836ed-9916-4935-a8f0-37029719281a)

[^1]: https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow
[^2]: https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow#running-a-workflow